cosima_cookbook/querying.py

```diff
@@ -310,6 +311,10 @@ def getvar(
             must be unique on the returned variables. Defaults to
             {'cell_methods': 'time: mean'} and should not generally be
             changed.
+    return_dataset - if True, return xarray.Dataset, containing the
+                     variable, along with extra variables like bounds
+                     otherwise, return xarray.DataArray containing only
+                     the variable
```

Comment:
The code is currently only checking for time bounds. It is probably good to be explicit here, so users know what to expect. It can (should) be changed if the functionality changes.

Reply:
Good point, fixed!